apps/cli/src/compose/rollups/docker-compose-espresso.yaml

```diff
@@ -14,36 +15,28 @@ services:
                 condition: service_healthy
         environment:
             ESPRESSO_SEQUENCER_L1_PROVIDER: ${CARTESI_BLOCKCHAIN_HTTP_ENDPOINT:-http://anvil:8545}
-            ESPRESSO_SEQUENCER_API_PORT: 8770
-            ESPRESSO_BUILDER_PORT: 8771
-            ESPRESSO_PROVER_PORT: 8772
+            ESPRESSO_SEQUENCER_API_PORT: 24000
+            ESPRESSO_BUILDER_PORT: 23000
             ESPRESSO_DEV_NODE_PORT: 20000
-            ESPRESSO_SEQUENCER_POSTGRES_HOST: database
-            ESPRESSO_SEQUENCER_POSTGRES_PORT: 5432
-            ESPRESSO_SEQUENCER_POSTGRES_USER: postgres
-            ESPRESSO_SEQUENCER_POSTGRES_PASSWORD: password
-            ESPRESSO_SEQUENCER_POSTGRES_DATABASE: espresso
             ESPRESSO_SEQUENCER_ETH_MNEMONIC: ${CARTESI_AUTH_MNEMONIC:-test test test test test test test test test test test junk}
-            ESPRESSO_SEQUENCER_L1_POLLING_INTERVAL: "1s"
-            ESPRESSO_STATE_PROVER_UPDATE_INTERVAL: "1s"
+            ESPRESSO_SEQUENCER_L1_POLLING_INTERVAL: "200s"
```

Comment:
is it ok?

Reply:
No, it should be 200ms think

Reply:
searching in their repo I found: 100ms, 200ms and 7s
<img width="694" alt="image" src="https://github.com/user-attachments/assets/8f3cfde1-b94f-4da9-8278-bce260eaca80" />

Reply:
sent a fixup at 0572649